script/deploy/safe/propose-to-safe.ts

```diff
@@ -106,29 +133,48 @@ const main = defineCommand({
       nonce: nextNonce,
     }
 
-    const safeTransaction = await protocolKit.createTransaction({
+    let safeTransaction = await protocolKit.createTransaction({
       transactions: [safeTransactionData],
     })
 
     const senderAddress = await signer.getAddress()
+    safeTransaction = await protocolKit.signTransaction(safeTransaction)
```

Comment:
nitpick: lines 136, 141, 142 are more related to each other so I would put them next to each other and move line 140 below them.